### DISCUSSION TITLE
how do i update jellyfin?

### DISCUSSION THREAD

**Discussion by bingobob** [tteck/Proxmox#2672] [General]
i type "update" within the LXC but get this?

root@jellyfin:~# update 
Command 'update' not found, did you mean:....

Do i run the 'install' script again from at host level ?

also - is this how it's generally done for other tteck script installed LXCs?

Thanks,

**Comment by tteck** [comment]
Jellyfin is updated via `apt update && apt upgrade`

**Comment by gabacode** [comment]
> i type "update" within the LXC but get this?
> 
> root@jellyfin:~# update Command 'update' not found, did you mean:....
> 
> Do i run the 'install' script again from at host level ?
> 
> also - is this how it's generally done for other tteck script installed LXCs?
> 
> Thanks,

That's how I did it:
```
1. apt remove jellyfin jellyfin-server jellyfin-web
2. rm /etc/apt/sources.list.d/jellyfin.sources
3. curl https://repo.jellyfin.org/install-debuntu.sh | sudo bash
4. jellyfin --version
```

**Reply by DanielOfir** [reply]
I've tried using the 'update' command, it said the LXC has been updated when in fact the version remained the same.
This worked for me, thanks!

**Reply by janas-karol** [reply]
> > i type "update" within the LXC but get this?
> > root@jellyfin:~# update Command 'update' not found, did you mean:....
> > Do i run the 'install' script again from at host level ?
> > also - is this how it's generally done for other tteck script installed LXCs?
> > Thanks,
> 
> That's how I did it:
> 
> ```
> 1. apt remove jellyfin jellyfin-server jellyfin-web
> 2. rm /etc/apt/sources.list.d/jellyfin.sources
> 3. curl https://repo.jellyfin.org/install-debuntu.sh | sudo bash
> 4. jellyfin --version
> ```

`apt update && apt upgrade` was not working for me but this helped, thanks!
